RadarSDK/Internal/RadarUtils.m

```diff
@@ -45,7 +45,7 @@ + (NSNumber *)timeZoneOffset {
 }
 
 + (NSString *)sdkVersion {
-    return @"3.3.0";
+    return @"3.3.1-beta.1";
```

Comment:
Example/Example.xcodeproj/project.pbxproj has a different number - is that ok?

Reply:
See above. As long as it matches the `.podspec`, we're good.